shell/browser/api/electron_api_session.cc

```diff
@@ -1010,7 +1138,8 @@ void SetSpellCheckerDictionaryDownloadURL(gin_helper::ErrorThrower thrower,
                                           const GURL& url) {
   if (!url.is_valid()) {
     thrower.ThrowError(
-        "The URL you provided to setSpellCheckerDictionaryDownloadURL is not a "
+        "The URL you provided to setSpellCheckerDictionaryDownloadURL is not "
+        "a "
```

Comment:
Is this change added by the linter? If so can you merge this line and the line below?